core/src/components/datetime/test/position/index.html

```diff
@@ -25,7 +25,7 @@
       <ion-content class="ion-padding">
         <ion-button id="open-datetime" onclick="presentPopover(defaultPopover, event)">Present Popover</ion-button>
         <ion-popover class="datetime-popover" id="default-popover">
-          <ion-datetime value="2022-05-03"></ion-datetime>
+          <ion-datetime value="2022-05-03T14:23:00.000Z"></ion-datetime>
```

Comment:
I made this change because of a minor regression where the hour column would jump down to 11 on opening the popover for the first time, on Firefox only. I decided this was acceptable because the missing time isn't a supported use case anyway, and the test is checking the position of the popover itself (which isn't affected). So I added a defined time to skirt the odd behavior.